.github/workflows/golang_lint_and_test.yml

```diff
@@ -8,14 +8,17 @@ jobs:
       fail-fast: false
     steps:
       - uses: actions/checkout@v2
-      - uses: codespell-project/actions-codespell@master
+      - name: Codespell
+        uses: codespell-project/actions-codespell@master
         with:
           ignore_words_list: "actualy,nwe"
           skip: "go.mod,go.sum"
-      - uses: actions/setup-go@v2
-      - run: go version
-      - uses: golangci/golangci-lint-action@v2
+      - name: Setup Go
+        uses: actions/setup-go@v2
+      - name: Run Golang CI Lint
+        uses: golangci/golangci-lint-action@v2
```

Comment:
I am not an active member of this Go repo so you guys decide.

But as a personal pet peeve, I tend not to `name:` job steps so that when they fail, I can quickly see the actual `uses:` or `run:` text rather than some vague English equivalent.

Reply:
Ah ok. I'm personally of the opinion that if the steps are orthogonal (or derived) then they should run under their own title to segregate between other steps and help with finding where things went wrong (and is beginner friendly imo)

Reply:
I do agree to small, discreet steps so that the contributor knows exactly which command failed.

Reply:
Looks like @tjgurwara99 named each command separately, not a number of commands together. So I think naming them is ok.